src/views/components/admin/deleteParkingLotForm.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react';
+import { Container, Form, Button, Alert } from 'react-bootstrap';
+import useDeleteParkingLot from '../../../controllers/useDeleteParkingLot.js';
+import useAdminGetParkingLots from '../../../controllers/useAdminGetParkingLot.js';
+
+function DeleteParkingLotForm() {
+    const [chosenLotID, setChosenLotID] = useState('');
+
+    const { handleParkingLotDeletion, error } = useDeleteParkingLot();
+    const { parkLots, fetchParkLots, parkLotError } = useAdminGetParkingLots();
+
+    const handleSelectLot = (e) => {
+        if (e.target.value !== "Choose..."){
+            const selectedValue = JSON.parse(e.target.value);
+            setChosenLotID(selectedValue.id);
+        } else{
+            setChosenLotID("Invalid");
+        }
+    };
+
+    const handleSubmit = async (event) => {
+        event.preventDefault();
+        const chosenLot = {chosenLotID};
+        await handleParkingLotDeletion(chosenLot);
+    };
+
+    useEffect(() => {
+        fetchParkLots();
```

Comment:
We want to fetch periodically, every 10 seconds for example. Check out the example in parkingRequestList component